docker/Dockerfile

```diff
@@ -85,6 +84,7 @@ EXPOSE 2181 2888 3888 8080
 ENV PATH=$PATH:/$DISTRO_NAME/bin \
     ZOOCFGDIR=$ZOO_CONF_DIR
 
+COPY test.py .
```

Comment:
should we give it a different name `check_status.py`

Reply:
Sure we can rename it.